docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+---
+id: data-download
+title: How to Download Data
+---
+
+The ARGO Data Platform uses the score-cient as a file download manager. Score facilitates the transfer of data with resumable download and built in BAM/CRAM slicing to make data download transfer fast and smooth.
+
+Please note:
+
+- Downloads are done in part and can be paused/resumed as needed
+- The score-client will automatically resume downloads if interrupted or paused briefly
+
+## Searching for Files
+
+Use the Platform [File Repository](https://platform.icgc-argo.org/repository) to search for a file set of interest. Query results can be narrowed down by selecting specific file values in the facet filters on the left side of the page. Once you have a file set identified, click `Download > File Manifest` on the top right of the table to download a TSV file manifest.
```

Comment:
suggestion for first sentences:
Platform users can search for a file set of interest using the File Repository. Files can be narrowed down by selecting specific values in the filter panel on the left side of the page.

(the interface says filters so would probably use that instead of facet)

Reply:
@kcullion yes probably should include screenshots, but wanted to wait till ui was a little more ready.

Reply:
i.e the query bar and stuff